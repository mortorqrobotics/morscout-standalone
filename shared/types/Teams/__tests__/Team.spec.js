import { loadTeams, updated } from "../index";

test("load Team stays the same", () => {
  expect(loadTeams).toMatchSnapshot();
});

test("updated Team stays the same", () => {
  expect(updated).toMatchSnapshot();
});
