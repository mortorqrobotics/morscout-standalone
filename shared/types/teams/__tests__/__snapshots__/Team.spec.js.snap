// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`load Team stays the same 1`] = `"server/getTeams"`;

exports[`updated Team stays the same 1`] = `"UPDATED_TEAMS"`;
